README.md

```diff
@@ -32,15 +32,23 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? Its a markup that gives meaning to web pages other than presentation. 
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+2. Name two big differences between ```display: block;``` and ```display: inline;```
+Display block takes up the whole width of the page. 
+Display Inline height & width propertis has no effect. 
```

Comment:
Great answers, but inline elements will have width properties. It's only top and bottom that have no effect.